files/build_templates/sonic_debian_extension.j2

```diff
@@ -263,6 +263,10 @@ fi
 sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/sonic-device-data_*.deb || \
     sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -f
 
+# Install cracklib  (and its dependencies via 'apt-get -y install -f')
```

Comment:
This HLD document section is not updated, I will push the updates soon. (I sent an email with those updates in the past.)
The updates are that the feature will be always compiled, and it will be disabled by default from running time POV. 
So the user can set enabled/disabled from running time only and not from compilation time.